rasa/utils/tensorflow/rasa_layers.py

```diff
@@ -901,6 +920,8 @@ def _features_as_token_ids(
                     self._tf_layers[self.SPARSE_TO_DENSE_FOR_TOKEN_IDS](f)
                 )
 
+        return None
```

Comment:
if it does return `None` then that case needs to be handled in the usage of this function which is `_create_mlm_tensors`.

Reply:
Will revert that one. That change was from merges with main.

Reply:
wait, this None is perfectly fine it just makes explicit what happens anyway